typescript/test/test_application/showcase-setup.sh

```diff
@@ -0,0 +1,22 @@
+mkdir showcase-server 
+cd showcase-server
+export GAPIC_SHOWCASE_VERSION=0.5.0
+export OS=`uname`
+export ARCH=amd64
```

Comment:
`uname -m`

Reply:
Well... nope, let's leave `amd64`. We don't publish binaries for other platforms anyway.